c/tskit/tables.c

```diff
@@ -4960,66 +4982,63 @@ tsk_table_sorter_sort_mutations_canonical(tsk_table_sorter_t *self)
 }
 
 static int
-tsk_table_sorter_sort_individuals(tsk_table_sorter_t *self)
+tsk_individual_table_count_descendants(
+    tsk_individual_table_t *self, tsk_size_t *num_descendants, tsk_id_t *traversal_order)
```

Comment:
This is nitpicking, but we'd normally put any optional arguments last - you'd never pass in NULL for traversal_order, right?